### PR TITLE
Use slf4j logging of parametrized messages with an exception

### PR DESCRIPTION
**Pull request by saalistaja**

### CODE REVIEW

File: enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/IdentityVerificationStatusService.java [hunk 1]
```diff
@@ -192,11 +192,11 @@ public IdentityVerificationStatusResponse checkIdentityVerificationStatus(Identi
                     identityVerificationFinishService.finishIdentityVerification(ownerId);
                 }
             } catch (DocumentVerificationException e) {
-                logger.error("Checking identity verification result failed, " + ownerId, e);
+                logger.error("Checking identity verification result failed, {}", ownerId, e);
                 response.setIdentityVerificationStatus(IdentityVerificationStatus.FAILED);
                 return response;
             } catch (OnboardingProcessException e) {
-                logger.error("Updating onboarding process failed, " + ownerId, e);
+                logger.error("Updating onboarding process failed, {}", ownerId, e);
```
Comment (banterCZ) [inline enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/IdentityVerificationStatusService.java:199]: Out of scope of this PR, but what do you think about writing `Updating onboarding process failed, ownerId={}`, IMHO it provides better context for readers.

**Review by banterCZ — APPROVED** [review]

File: enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/IdentityVerificationStatusService.java (hunk 1 above)
Comment (saalistaja) [reply enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/IdentityVerificationStatusService.java:199]: Well it is a way but with repeated words. The logged message would look like `ownerId=OwnerId(activationId=123,userId=456)` which I am not fan of.

Comment (banterCZ) [reply enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/IdentityVerificationStatusService.java:199]: Oh, I see, you are right, I have missed that `ownerId` is not a primitive.

**Review by romanstrobl — APPROVED** [review]
Nice improvement!

**Pull request merged**